deploy-summary/api/ui.js

```diff
@@ -9,7 +9,7 @@ module.exports = withUiHook(async ({ payload }) => {
 
   const record = await store.findOne({ ownerId })
 
-  const providers = ['github', 'gitlab']
+  const providers = ['GitHub', 'GitLab']
```

Comment:
Line 15, 16, and 21 are affected by this change and should be updated.

We might want to update this to:
```suggestion
  const providers = [{ id: 'github', name: 'GitHub' }, { id: 'gitlab', name: 'GitLab' }]
```
so we can keep the logic clearer in the code.